docs-js/release-notes.mdx

```diff
@@ -0,0 +1,175 @@
+---
+id: release-notes
+title: Release Notes
+sidebar_label: Release Notes
+description: Release notes of SAP Cloud SDK for AI, stay up to date with the recent features, fixes, dependency updates and recommendations.
+keywords:
+  - sap
+  - cloud
+  - sdk
+  - ai
+---
+
+## Should I update?
+
+We highly recommend regularly updating to the latest SAP Cloud SDK for AI version.
```

Comment:
I will not add this, seems unnecessary. I dont think we had anything in the cloud sdk docs